src/Bundle/CoverallsBundle/Api/Jobs.php

```diff
@@ -132,7 +132,15 @@ public function send()
 
         $jsonPath = $this->config->getJsonPath();
 
-        return $this->upload(static::URL, $jsonPath, static::FILENAME);
+        $url = static::URL;
+
+        $endpoint = getenv('COVERALLS_ENDPOINT');
+
+        if ($endpoint) {
+            $url = $endpoint . '/api/v1/jobs';
```

Comment:
This is the ` '/api/v1/jobs'` const will be changed in the future.

We should let `$endpoint` be the whole request URL and we can change into this `COVERALLS_ENDPOINT` be the endpoint URL.

Reply:
All of our other integration libraries expect `COVERALLS_ENDPOINT` to only define the hostname/protocol, e.g. "https://coveralls.your-internal-network.com", so we'd prefer to keep the path '/api/v1/jobs' separate.

Reply:
@nickmerwin, thank you for your explanation.

I think we can let the `'/api/v1/jobs'` path be another environment variable.

And this will change the version path easily.

What do you think about that?

Reply:
@peter279k that sounds great, thank you!